lib/voomex/rapidsms.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Voomex.RapidSMS do
+  @moduledoc """
+  Handles communication to RapidSMS
+  """
+
+  alias Ecto.UUID
+
+  @doc """
+  Send the PDU to RapidSMS
+  """
+  def send_to_rapidsms(pdu) do
+    pdu
+    |> pdu_to_request
+    |> Voomex.RapidSMS.Worker.new()
+    |> Oban.insert()
+  end
+
+  def pdu_to_request(pdu) do
+    config = Application.get_env(:voomex, Voomex.SMPP)
+
+    %{
+      content: pdu.mandatory.short_message,
+      from_addr: pdu.mandatory.source_addr,
+      group: nil,
+      in_reply_to: nil,
+      message_id: UUID.generate(),
```

Comment:
If I'm understanding correctly, I guess this is where #15 might come in at some point?